src/test/regress/citus_tests/test/test_other_databases.py

```diff
@@ -5,6 +5,9 @@ def test_main_commited_outer_not_yet(cluster):
     # create a non-main database
     c.sql("CREATE DATABASE db1")
 
+    c.sql("ALTER SYSTEM SET citus.local_hostname TO '127.0.0.1'")
+    c.sql("SELECT pg_reload_conf()")
```

Comment:
This should not be necessary. In production environments it's expected that citus.local_hostname is not the same as the value in pg_dist_node. So if the tests require that to pass, then that's a big problem (because that means they would fail in production environments).